backend/app/views/morpheus.ejs

```diff
@@ -15,7 +15,7 @@
       href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
     />
     <%- include('favicons'); %>
-    <title>Matrix - Morpheus</title>
```

Comment:
What do you think to maintain the #matrix - morpheus?

Reply:
Better!